pkgs/development/python-modules/vector/default.nix

```diff
@@ -21,17 +27,31 @@ buildPythonPackage rec {
     hatch-vcs
     hatchling
   ];
+
   propagatedBuildInputs = [
     numpy
+    packaging
+  ];
+
+  checkInputs = [
+    awkward
+    papermill
+    pytestCheckHook
   ];
 
-  checkInputs = [ pytestCheckHook ];
+  pythonImportsCheck = [
+    "vector"
+  ];
 
-  pythonImportsCheck = [ "vector" ];
+  disabledTestPaths = [
+    # jupyter_client.kernelspec.NoSuchKernel: No such kernel named python3
```

Comment:
Sinc you add the dependency on `papermill`, might as well fix the test:
```diff
diff --git a/pkgs/development/python-modules/vector/default.nix b/pkgs/development/python-modules/vector/default.nix
index e01394a5637..1c8a7d4d86d 100644
--- a/pkgs/development/python-modules/vector/default.nix
+++ b/pkgs/development/python-modules/vector/default.nix
@@ -4,9 +4,11 @@
 , fetchPypi
 , hatch-vcs
 , hatchling
+, numba
 , numpy
 , packaging
 , papermill
+, notebook
 , pytestCheckHook
 , pythonOlder
 }:
@@ -36,6 +38,8 @@ buildPythonPackage rec {
   checkInputs = [
     awkward
     papermill
+    numba
+    notebook
     pytestCheckHook
   ];
 
@@ -43,10 +47,7 @@ buildPythonPackage rec {
     "vector"
   ];
 
-  disabledTestPaths = [
-    # jupyter_client.kernelspec.NoSuchKernel: No such kernel named python3
-    "tests/test_notebooks.py"
-  ];
+  __darwinAllowLocalNetworking = true;
 
   meta = with lib; {
     description = "Library for 2D, 3D, and Lorentz vectors, especially arrays of vectors, to solve common physics problems in a NumPy-like way";
```

Reply:
Thanks, added.